jobs/config_test.py

```diff
@@ -441,13 +441,23 @@ def test_valid_timeout(self):
 
     def test_valid_job_config_json(self):
         """Validate jobs/config.json."""
+        ignore = [
+            'fake-failure',
+            'fake-branch',
+            'fake-pr',
+            'random_job',
+        ]
```

Comment:
Are these defined elsewhere?

Reply:
places like https://github.com/kubernetes/test-infra/blob/master/jenkins/bootstrap_test.py#L1287...

Reply:
Maybe leave a comment about keeping this in sync then? We should probably define this at most once per test script and make sure they're the same.

Reply:
added a comment - seems this is the only test cares about json validation though.